lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -55,6 +59,7 @@ def compute(self, student, course_structure, scores_client, submissions_scores):
             # self.scores may hold outdated data, force it to refresh on next access
             lazy.invalidate(self, 'scores')
 
+        log.info("accessed lazy property scores with value: {0}".format(unicode(self.scores)))
```

Comment:
On edge, we wouldn't have access to these log statements anyway - since they are at the `info` level.
In my mind, I consider access to these internal properties as internal logic that doesn't require monitoring at the macroscopic logging level.